loco_translate.module

```diff
@@ -16,3 +19,64 @@ function loco_translate_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_cron().
+ *
+ * Automation of Loco Translate module.
+ */
+function loco_translate_cron() {
+  $now = time();
+  $config = \Drupal::config('loco_translate.settings');
+  $utility = \Drupal::service('loco_translate.utility');
+
+  // Get configured langcode to pull from loco.
+  $langcodes = $config->get('automation.pull.langcodes');
+  if ($langcodes) {
+    foreach ($langcodes as $langcode => $status) {
+      // Skip unchecked langcode.
+      if (!$status) {
+        continue;
+      }
+
+      // Check last run of this langcode.
+      $last_run = $utility->getLastPull($langcode);
+
+      if ($now - $last_run >= $config->get('automation.pull.interval')) {
```

Comment:
Even if I think we don't have an issue with `Never` (interval value = 0), it would be more readable a more safe to handle this specific case. Wdyt?